tools/sa_requester/lib/index.js

```diff
@@ -0,0 +1,45 @@
+const command = require('@oclif/command')
+const fs = require('fs')
+// Can't abort fetch on nodejs which results in a process that never exits
+// on network request timeout. Use axios instead until support is added to fetch.
+const axios = require('axios')
+
+class SaRequester extends command.Command {
+  async run () {
+    const { args, flags } = this.parse(SaRequester)
+    const agreement = fs.readFileSync(flags.agreement, 'utf8')
+    const oracleURLs = args.file.split(/\s+/)
+
+    createServiceAgreements(agreement, oracleURLs)
+      .then(signatures =>
```

Comment:
It needs to log a message for success & failures. I feel like it's more clear to use `then` instead of something like.

```javascript
const signatures = createServiceAgreements(agreement, oracleURLs).catch(e => console.log('Unable to create SA, got error:\n\n\t%s\n', e.message))

if (signatures) {
   console.table(
      signatures.map(s => ({signature: s})),
      ['signature']
   )
}
```

Reply:
Have `createServiceAgreements` return an empty array rather than nil.
```js
try {
  const signatures = await createServiceAgreements(agreement, oracleURLs)
  console.table(signatures.map(s => {signature: s}), ['signature'])
} catch (e) {
  console.log('Unable to create SA, got error:\n\n\t%s\n', e.message)
}
```

Again though, this is optional and subjective so your call.